bliss/cfdp/test/pdu_test.py

```diff
@@ -21,7 +22,9 @@
 from bliss.cfdp.primitives import ConditionCode
 
 
-TEST_DIRECTORY = bliss.config.get('dsn.cfdp.test.path')
+TEST_DIRECTORY = bliss.config.get('dsn.cfdp.datasink.test.path')
```

Comment:
I think we can drop this configurable test directory option entirely and just create the test folder path off the `__file__`. Is there a use case that we're hoping to cover by allowing the user to configure the test data path? If not, let's avoid exposing that to users

Reply:
Nope no reason. Will update.